bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/mascot/MascotCat.scala

```diff
@@ -49,10 +51,11 @@ object MascotCat {
                        centerRA: Double, centerDec: Double,
                        factor: Double = Mascot.defaultFactor,
                        progress: ProgressFunction = defaultProgress,
-                       filter: Star => Boolean = Mascot.defaultFilter)
+                       filter: Star => Boolean = Mascot.defaultFilter,
+                       asterismPreFilter: List[SiderealTarget] => Boolean = Mascot.defaultAsterismPreFilter)
```

Comment:
Is there ever any other "pre filter" besides the magnitude check?  I guess it's kind of nice to have more flexibility I'm just wondering if we need it.  We end up having to pass this around a lot.

Reply:
No, this is the only check.

From my understanding, we wouldn't want to do a check when looking for ODGW guide stars, but we aren't using those anymore, correct?

Reply:
Right, I see why you did that then.  Let's just leave it this way for now because I believe Bryan wants to leave the door open for GSAOI asterisms in the future.